src/acquisition/covid_hosp/state_daily/database.py

```diff
@@ -0,0 +1,84 @@
+# first party
+from delphi.epidata.acquisition.covid_hosp.common.database import Database as BaseDatabase
+from delphi.epidata.acquisition.covid_hosp.common.utils import Utils
+
+
+class Database(BaseDatabase):
+
+  TABLE_NAME = 'covid_hosp_state_timeseries'
+
+  # These are the names that appear in the CSV header, in order of appearance
+  # in the database table, along with corresponding data type converters.
+  # However, note that the corresponding database column names may be shorter
+  # due to constraints on the length of column names. See
+  # /src/ddl/covid_hosp.sql for more information.
+  ORDERED_CSV_COLUMNS = [
```

Comment:
```suggestion
  # Additionally, all column names below are shared with state_timeseries,
  # except for reporting_cutoff_start (here) and date (there). If you need
  # to update a column name, do it in both places.
  ORDERED_CSV_COLUMNS = [
```

& add a matching comment in state_timeseries

Reply:
added to both